src/core/cm/storagestate/itf/storage.hpp

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2025 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_CORE_CM_STORAGESTATE_ITF_STORAGE_HPP_
+#define AOS_CORE_CM_STORAGESTATE_ITF_STORAGE_HPP_
+
+#include <core/common/types/types.hpp>
+
+namespace aos::cm::storagestate {
+
+/** @addtogroup cm Communication Manager
+ *  @{
+ */
+
+/**
+ * Instance info.
+ */
+struct InstanceInfo {
+    InstanceIdent                         mInstanceIdent;
```

Comment:
Should it be obsolete?
```suggestion
    InstanceIdentObsolete                         mInstanceIdent;
```